media/system/js/share-uncompressed.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @package         Joomla.JavaScript
+ * @copyright       Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+/**
+ * Calls the sending process of the config class
+ */
+jQuery(document).ready(function ($)
+{
+    $('#sharebutton').click(function ()
+    {
+        var share_draft = {
+            articleId  : $().val(),
```

Comment:
@nikitadhiman` $('#jform_id').val()`

Just view the field on the publishing tab with Firebug (or any other dev tool).

<img width="769" alt="screenshot 2016-07-06 19 04 57" src="https://cloud.githubusercontent.com/assets/897638/16627220/8feba462-43ac-11e6-989c-572f4c11e36e.png">